### DISCUSSION TITLE
Can't log in because it says "Your account has not been verified.

### DISCUSSION THREAD

**Discussion by SuperSandyman** [danny-avila/LibreChat#3562] [Help Wanted]
I don't know what it is, but when I try to log in, it says "Your account has not been verified. Please check your email for a verification link." 

What should I do 🤔

**Comment by berry-13** [comment]
if you haven't setup an email server, you need to disable email verification

```
ALLOW_UNVERIFIED_EMAIL_LOGIN=true
```

**Comment by danny-avila** [comment]
Is this your own instance? Did you setup email verification? If not, you should be able to login even with a message like that. You can also disable email verification, but you shouldn’t need to if email signups are not configured. The relevant env variable is `ALLOW_UNVERIFIED_EMAIL_LOGIN=TRUE`

The exact logs when you try to login would be helpful. If you are using docker, they are in the logs directory at project root

**Comment by SuperSandyman** [comment]
I was able to log in successfully 😀 Thank you!

**Comment by mendelgordon** [comment]
I'm having the same issue - the user was imported from the docker in my local machine to huggingface, so that may be what caused it - here's the logs I see:
2024-10-30 18:40:25 error: [resendVerificationEmail] Error resending verification email: connect ECONNREFUSED 127.0.0.1:25
{"level":"error","message":"Passport Local Strategy - Email not verified","parameters":[{"name":"email","value":"***@gmail.com"}],"timestamp":"2024-10-30T18:52:23.808Z"}
{"level":"error","message":"[Login] [Login failed] [Username: ***@gmail.com] [Request-IP: ***]","timestamp":"2024-10-30T18:52:23.809Z"}

EXECUTING: (requireLocalAuth) Error: Email not verified.

**Comment by mendelgordon** [comment]
Adding the variable to allow logging in without registration fixes it for me 
<img width="1236" alt="image" src="https://github.com/user-attachments/assets/cbd3f663-81ff-417e-952d-6e5f5d199e68">
